tvp_go19.go

```diff
@@ -36,6 +41,11 @@ type TVP struct {
 	Value interface{}
 }
 
+type indexPosition struct {
+	index    int
```

Comment:
The code looks good, but it could use a bit more docs. Even an internal comment describing the index of what and position of what would be nice.

Reply:
I commented